libs/designsystem/src/lib/components/page/page.component.spec.ts

```diff
@@ -71,11 +96,32 @@ describe('PageComponent', () => {
   });
 
   it('should render back button with correct size', async () => {
+    spectator.detectChanges();
     await TestHelper.whenReady(ionToolbar);
     const ionBackButton = spectator.queryHost('ion-toolbar ion-buttons ion-back-button');
     expect(ionBackButton).toHaveComputedStyle({
       width: fatFingerSize,
       height: fatFingerSize,
     });
   });
+
+  it('should hide tab bar when hideTabs is true on enter', () => {
+    spectator.component.hideTabs = true;
```

Comment:
You are calling `tabbar.hide()` in `hideTabs` setter, so this doesn't assert that the `onEnter` method hid the tabs...